### DISCUSSION TITLE
Topics getting recreated after deletion

### DISCUSSION THREAD

**Discussion by opsocomusr** [strimzi/strimzi-kafka-operator#5582] [Q&A]
We are deleting the kafka topic resource using `kubectl delete kafkatopic TOPIC-NAME` but the topics are automatically recreated after some time. We have the settings `delete.topic.enable: true` and `auto.create.topics.enable: false` in the broker configuration.

Why are the topics recreated even after manual deletion of all topics? Is there any configuration that will disable the topics from getting recreated?

**Comment by scholzj** [comment]
Well, it depends on who is recreating them. It could be a bug in the topic operator, but it also could be your apps recreating them. I guess a good start might sharing a debug log from the Topic Operator when it is happening.

**Reply by Piboonsak** [reply]
So how we know who create them. Because I already delete the connector that auto created topic but there still comeback.
Do you have soloution for this?
